content/applications/sales/point_of_sale/online_food_delivery/urban_piper.rst

```diff
@@ -0,0 +1,182 @@
+      create store process can take 2-3 minutes to update the locations on the UrbanPiper Atlas
+      platform. Store name is same as your point of sale name.
+
+Configure products for Online Food Delivery
+-------------------------------------------
+
+#. Go to the :guilabel:`Point of Sale` tab on the product form by navigating to
+   :menuselection:`Point of Sale --> Products --> Products`, then click on any product.
+
+#. Fill in the details under the :guilabel:`URBAN PIPER` section.
+
+    .. image:: urban_piper/product_form.png
+        :alt: Product form
+
+#. To make multiple products available for food delivery, switch to list view, select the products,
+   and Point of Sale :guilabel:`Available on Food Delivery`.
+
+    .. image:: urban_piper/product_list.png
+        :alt: Product list
+
+Sync Menu
+=========
+
+#. Go to :ref:`POS settings <configuration/settings>` and find the Food Delivery Connector.
+#. Press the :guilabel:`Sync menu` button.
+
+    .. note::
+      After successfully syncing the menu, a success notification will appear. Below the
+      :guilabel:`Sync menu` button, the :guilabel:`Last Sync on` timestamp will display. Note that
+      the sync menu process can take 2-3 minutes to update the menu on the UrbanPiper Atlas
+      platform.
+
+    .. image:: urban_piper/sync_menu.png
+        :alt: Sync menu
+
+Request to Go Live
+==================
+
+#. Log-in in `Go to the Locations <https://atlas.urbanpiper.com/locations>`_ tab.
+
+    .. image:: urban_piper/atlas_location.png
+        :alt: Locations menu
+
+#. Click on the location you want to activate, then press the :guilabel:`Request to Go Live` button.
+
+    .. image:: urban_piper/location_go_live.png
+        :alt: Go live
+
+#. Select the platform(s) you want to activate and press Next.
+
+    .. image:: urban_piper/go_live_popup.png
+        :alt: Go live popup
+
+#. Now, configure the platform’s parameters, such as Platform ID and Platform URL, to establish the
+   connection between the platform and UrbanPiper, then press the :guilabel:`Request to Go Live`
+   button.
+
+    .. image:: urban_piper/go_live_parameters.png
+        :alt: Go live parameters
+
+#. To verify that your location is live, check the list view of locations. In the
+   :guilabel:`Assoc. platform(s)` column, click on any provider and review the status of that
+   specific platform for this location.
+
+    .. image:: urban_piper/platform_status.png
+        :alt: Platform status
+
+Order Flow
+==========
+
+#. When someone places an order via any food delivery platform :guilabel:`i.e. Zomato, UberEats`,
+   you will be notified with sound and a notification. To view the order, simply click on
+   "Review Order" and you will be redirected to the orders page.
+
+    .. image:: urban_piper/order_notification.png
+        :alt: Order notification
+
+#. Additionally, there is a cart button in the navbar. Clicking this button will give you options
```

Comment:
```suggestion
#. Additionally, there is a cart button in the navbar. Clicking this button gives you options
```
"gives you options" isn't clear? "_The number of orders at each stage is displayed_" or something like that?